.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: python
+python:
+  - '2.7'
+install:
+  - npm install -g dredd
+  - pip install -r requirements.pip coveralls coverage
+script:
+  - "coverage run --source='.' ./manage.py test"
+  - ./manage.py migrate
```

Comment:
It's not quite that easy. The `dredd_hooks` script needs to access to the
django context to create models. I don't know of a reasonable way to have
it hook into a test db setup, and I did a bit of research on it.

I think the longer term fix would be to define a separate testing settings
file with a different db config and provide helpers to activate that for
running tests.

On Tue, Nov 10, 2015 at 3:05 PM, George Schneeloch <notifications@github.com
> wrote:

> In .travis.yml
> <https://github.com/mitodl/ccxcon/pull/7#discussion_r44457405>:
>
> > @@ -0,0 +1,14 @@
> > +language: python
> > +python:
> > +  - '2.7'
> > +install:
> > +  - npm install -g dredd
> > +  - pip install -r requirements.pip coveralls coverage
> > +script:
> > +  - "coverage run --source='.' ./manage.py test"
> > +  - ./manage.py migrate
>
> It would be nice to move this code outside travis.yml to allow local
> testing. Would it be possible to install tox and move this to tox.ini?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mitodl/ccxcon/pull/7/files#r44457405>.
>